gradle.properties

```diff
@@ -6,8 +6,8 @@ POM_URL=https://github.com/airbnb/lottie-android
 POM_SCM_URL=https://github.com/airbnb/lottie-android
 POM_SCM_CONNECTION=scm:git@github.com:airbnb/lottie-android.git
 POM_SCM_DEV_CONNECTION=scm:git@github.com:airbnb/lottie-android.git
-POM_LICENSE_NAME=Apache License 2.0
-POM_LICENSE_URL=https://github.com/airbnb/lottie-android/blob/master/LICENSE
+POM_LICENSE_NAME=Apache-2.0
+POM_LICENSE_URL=https://www.apache.org/licenses/LICENSE-2.0.txt
```

Comment:
Licensee should support this. This is a common pattern. I've never use it myself but could you double check?

Reply:
The canonical Apache 2 URL is specified by [the SPDX database](https://spdx.org/licenses/Apache-2.0.html) as https://www.apache.org/licenses/LICENSE-2.0, but https://www.apache.org/licenses/LICENSE-2.0.txt is explicitly supported as a fallback URL: https://github.com/cashapp/licensee/blob/9281c96d83d664ec8c39ba0a0adac3bf7bfccc44/src/main/kotlin/app/cash/licensee/licenses.kt#L64

Reply:
Thanks for the info, @JakeWharton. So is the recommended approach the full proposal here minus `.txt`?

Reply:
I would feel more confident saying yes if the POM format had a statement that using the canonical SPDX URL was recommended, but both of these Apache URLs are so ubiquitous that I don't think it matters all that much.

Reply:
I changed it to the non txt one as the apache license states:

> You may obtain a copy of the License at
> 
> http://www.apache.org/licenses/LICENSE-2.0

They really should recommend using https for obtaining their license 🙄